scripts/should_rebuild.js

```diff
@@ -1,8 +0,0 @@
-'use strict'
```

Comment:
With the new logic I think it's fine to remove this file. However, we should cache the `build` folder in most cases where we're not explicitly testing if building works properly to avoid unnecessary rebuilds.